packages/design-system/src/themes/ceramic-light/colors/ceramics.ts

```diff
@@ -1,27 +1,22 @@
 import { Palettes } from './palettes'
 import { Atomics } from './atomics'
+import triangleCompensationSvg from './triangleCompensation.svg'
```

Comment:
suggested filename for assets: `kebab-case`.

approved, but you can change these filenames later in another PR (same applies to `ceramicBg.webp`/etc.)